platform.md

```diff
@@ -1074,7 +1095,11 @@ For more information on build reproducibility see [https://reproducible-builds.o
 
 Where:
 - `previous-image.reference` MUST be either a digest reference to an image in an OCI registry or the ID of an image in a docker daemon
+  - In case an image in [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format is being used, it will also include the path of the image in OCI layout format following the rules describe [previously](#how-to-map-an-image-reference-into-a-path-in-the-layout-repository) 
+  - The format MUST be as follows: `[path]@[digest]`  
```

Comment:
```suggestion
```

I can't suggest edits to the line above as it didn't change, but I would put:

```
- `previous-image.reference` MUST be either:
  - A digest reference to an image in an OCI registry
  - The ID of an image in a docker daemon
  - The path to an image in OCI layout format
```

Why does it say `[path]@[digest]` when the mapping rules state `Lifecycle will load the image from disk in [OCI Image Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format at `<layout-dir>/<registry>/<repo>/<image>/<algorithm>/<digest>` (digest is part of the path)

Reply:
Well, remember previously I had a field `Name`? I removed that one in order to keep the existing one `Reference`, the reference is filled with the `Image.Identifier()` method. because, OCI layout requires the path I changed the implementation of `layout.Image.Identifier()` according to the talk we had in this [thread](https://cloud-native.slack.com/archives/C0331B5QS02/p1674761476209019)

Reply:
Definitly the digest is not required, even thought it can be use to validate the correctness of the run-image, I added because the other two cases (daemon, registry) had some kind of ID